deploy/crds/picchu.medium.engineering_clusters_crd.yaml

```diff
@@ -147,10 +146,9 @@ spec:
               - version
               type: object
           type: object
+      type: object
   version: v1alpha1
-status:
```

Comment:
there's something about this, i can't remember exactly, but status should be stored as a separate object internally and this might be how that's controlled? I don't remember, just look for status fetch/save errors in picchu to make sure it's working after your change.

Reply:
I did see some errors ("the server could not find the requested resource..."). They seemed to resolve after I added some `+kubebuilder` annotations to the `_types.go` files, and I applied the generated (and unmodified) CRD files. 🤞